sockets/arcade.js

```diff
@@ -0,0 +1,23 @@
+module.exports = function (io) {
+  var data = [];
```

Comment:
Want to make this an object by default? Since that's what it's going to end up being